packages/components/src/internal/components/editable/EditableGridPanel.tsx

```diff
@@ -89,7 +133,10 @@ export const EditableGridPanel: FC<Props> = memo(props => {
             onChange={_onChange}
             queryInfo={queryInfo}
             readonlyRows={activeReadOnlyRows}
+            readOnlyColumns={readOnlyColumns}
             updateColumns={activeUpdateColumns}
+            allowExport={true}
```

Comment:
does this need the extra `allowExport` prop or can the existence of the `exportHandler` suffice?